src/components/Book.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import { removeBookAsync } from '../redux/books/books';
+import 'react-circular-progressbar/dist/styles.css';
+
+const Book = (props) => {
+  const {
+    id, title, author, category,
+  } = props;
+  const dispatch = useDispatch();
+  const progress = parseInt(Math.random() * 100, 10);
+  return (
+    <div className="book" id={id}>
+      <div className="bookInfo">
+        <span className="bookCategory">{category}</span>
+        <span className="bookTitle">{title}</span>
+        <span className="bookAuthor">{author}</span>
+        <div className="bookActions">
+          <button type="button">Comments</button>
+          <div className="verticalLine" />
+          <button
+            type="button"
+            onClick={() => {
+              dispatch(removeBookAsync(id));
+            }}
+          >
+            Remove
+          </button>
+          <div className="verticalLine" />
+          <button type="button">Edit</button>
+        </div>
+      </div>
+      <div className="infoContainer">
+        <div className="progressContainer" />
+        <div className="verticalLine big">
+          <CircularProgressbar value={progress} text={`${progress}%`} />
+        </div>
+        <div className="chapterInfo">
+          <div className="currenChapter">chapter info</div>
+          <div>introduction</div>
+          <button type="button">Update progress</button>
+        </div>
+      </div>
+    </div>
+  );
```

Comment:
- [ ] Kindly make sure your book item block design sticks as much as possible to the template design for the project requirement.
### Template design
![Screenshot from 2022-11-17 21-26-19](https://user-images.githubusercontent.com/83042742/202541265-ca8fdffc-6925-423f-82a5-5b795d0cfb71.png)

### You design
![Screenshot from 2022-11-17 21-24-01](https://user-images.githubusercontent.com/83042742/202541415-e5e53c26-5572-46a2-bd42-e883b805084e.png)